tests/queries/0_stateless/01408_range_overflow.sql

```diff
@@ -1,12 +1,13 @@
 -- executeGeneric()
-SELECT range(1025, 1048576 + 9223372036854775807, 9223372036854775807); -- { serverError 69; }
```

Comment:
Nice catch! 
That test was told us that there is a bug in the old realisation.

1025 and 9223372036854776832 are UInt64.
The next value is 18446744073709552639 and it is out of range UInt64. But that value do not present in result and shouldn't affect the result.

Now the behaviour is better.

Reply:
> 

@CheSema thank you